experimenter/manifesttool/version.py

```diff
@@ -1,14 +1,18 @@
+from __future__ import annotations
+
 import plistlib
 import re
 from dataclasses import dataclass
-from typing import Iterable, Optional
+from typing import Any, Iterable, Optional, TYPE_CHECKING
 
 from requests import HTTPError
 
 from manifesttool import github_api, hgmo_api
-from manifesttool.appconfig import AppConfig, RepositoryType, VersionFile, VersionFileType
 from manifesttool.repository import Ref
 
+if TYPE_CHECKING:
+    from manifesttool.appconfig import AppConfig, VersionFile
```

Comment:
Why not just import these normally? I'm not a fan of conditional imports 🤔

Reply:
Usually something like this is to avoid circular imports, see here:

https://mypy.readthedocs.io/en/stable/runtime_troubles.html#import-cycles

Reply:
That makes sense 👍